src/pages/scheduler/components/Toolbar.tsx

```diff
@@ -14,11 +14,27 @@ export const CalendarToolBar =
         onSelectedDateChange(new Date());
       }
     };
+
+    const handleDownload = () => {
+      if (vCalendar) {
+        const element = document.createElement('a');
+        const file = new Blob([vCalendar], { type: 'text/plain' });
+        element.href = URL.createObjectURL(file);
+        element.download = 'calendar.ics';
+        document.body.appendChild(element); // Required for this to work in FireFox
+        element.click();
+        document.body.removeChild(element);
+      }
+    };
+
     return (
       <Flex pb="0.5em" justifyContent="space-between" alignItems="center">
         <Heading size="md">Scheduler</Heading>
         <Text fontSize="xl">{label}</Text>
         <HStack pb="0.2em">
+          <Button size="sm" colorScheme="gray" onClick={handleDownload} disabled={!vCalendar}>
```

Comment:
nit: maybe a download icon or maybe a colour that stands out a bit, like blue?

Reply:
done